### PR TITLE
W-14689654 acb int exchange duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 1]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
...
 +
 image::int-empty-canvas.png["Canvas showing visual representation of Mule flow and the Mule configuration file"]
 +
 [calloutlist]
 .. The canvas provides space for a visual representation of your Mule flows or subflows.
 .. The configuration XML editor displays the configuration file for your Mule application.
-. Select *Flow* from the canvas to create an empty flow.
-. In the configuration XML, replace the name of the flow `name1` with your flow name, for example, `getFlights`.
+. Select *Build a Flow* from the canvas to create an empty flow within a Mule integration application.
+. Change the default name of the flow by using one of these procedures:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]: I wonder if these two procedures (using config UI vs. config XML) should live in two tabs? I initially thought that both procedures are required, which wouldn't be correct since modifying the default name in the config UI automatically updates the XML. I had to go back and re-read the step to see it said "using **one** of these procedures:". My bad, but I might not be the only one to make this mistake. Tabs might be clearer.

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 2]
```diff
@@ -87,7 +127,7 @@ For example, add the `<http:listener-config/>` component from a snippet.
 +
 For more information about snippets, see xref:int-work-with-code-snippets.adoc[].
 +
-.. In the configuration XML, place your cursor before the `<flow/>` element and type `http`.
+.. In the configuration XML, place your cursor above the opening `<flow>` tag, and type `http`.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]: Our style guide says that "above" is positional and therefore not accessible. But tbh I don't know if there's a workaround, so it's probably moot.

Also, if I'm understanding correctly, the meaning changes from:

```
<flow>
place cursor here
</flow>
```

To:

```
place cursor here
<flow>
</flow>
```

Is this correct?

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 3]
```diff
@@ -25,7 +27,7 @@ to process your business logic, and configure attributes:
 * xref:start-acb.adoc[Set up and access the web or desktop IDE].
 
 [[create-integration-project]]
-== Create a New Integration Project
+== Create an Integration Project
 
 To create a new integration project:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
To create an integration project:
```
Should this be consistent with the H2 title?

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 4]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
 
 include::partial$acb-reusable-steps.adoc[tags="proj-directory-warn"]
 |===
-. Click *Create Project*.
+. Opt to create a project from scratch or to start with an asset from Anypoint Exchange:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
. Create a project from scratch, or start with an asset from Anypoint Exchange:
```
This is maybe a little simpler?

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 5]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
...
+
+* *Empty Project*
++
+Start an integration project from scratch, without any preconfigured components.
++
+image::int-create-integration-scratch.png["Create integration project from scratch.", width=70%]
+
+* *Template or Example Project*
++
+Import an existing integration template or example from Anypoint Exchange.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]: Nit picky: 
I'm confused a little here (probably due to my lack of product knowledge.)

Is an integration template the same as an integration example? Do both really need to be mentioned? The sentence can be interpreted as saying they are the same thing. I notice the product UI mentions both, so perhaps the customer already knows what's up.

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 6]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
...
++
+image::int-create-integration-scratch.png["Create integration project from scratch.", width=70%]
+
+* *Template or Example Project*
++
+Import an existing integration template or example from Anypoint Exchange. 
++
+image::int-create-integration.png["Create integration project from an Exchange asset."]
++
+If you opt to start with an asset from Exchange:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
If you start with an asset from Exchange:
```

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 7]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
...
 
+.. xref:start-acb.adoc#login-ide[Log in to Anypoint Platform from the IDE] so that you can find assets.
+.. From the IDE, search for the template or example.
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]: See comment above (line 65). The sentence can be read as templates are the same as examples. At least this is my read.

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 8]
```diff
@@ -109,37 +149,44 @@ For example, add a Set Payload component to your HTTP Listener operation:
 +
 .. In the canvas, click the image:icon-plus.png["",15,15] (*Add component*) icon.
 .. In the *Add Component* panel, search for and select *Set payload* from the *Transformer* results.
+.. In the canvas, click Set Payload to open its configuration UI, and add a  value between the square brackets that appear `#[]`:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
.. In the canvas, click *Set payload* to open its configuration UI, and add a value between the square brackets that appear (`#[]`):
```
Consistency with previous line (line 151), which doesn't capitalize "payload", and small suggestion to put the square brackets in parentheses.

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 9]
```diff
@@ -109,37 +149,44 @@ For example, add a Set Payload component to your HTTP Listener operation:
...
+
+* To add a string, type a value, such as `my value`, for example:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
* To add a string, type a value such as `my value`. For example:
```
This is maybe a little simpler?

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 10]
```diff
@@ -109,37 +149,44 @@ For example, add a Set Payload component to your HTTP Listener operation:
...
+</flow>
 --
 
 Notice that the Listener component in the canvas now shows an error:
 
 image::int-canvas-error.png["Listener error in the canvas"]
 
 To determine where the error is, select the processor in the canvas.
-Anypoint Code Builder highlights its location within the configuration XML, for example:
+Anypoint Code Builder highlights its location within the configuration XML, and you can mouse over the issue for more information, for example:
```
Comment (chris-campbell9) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
Anypoint Code Builder highlights its location within the configuration XML, and you can mouse over the issue for more information. For example:
```

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 2 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: I think it always meant to place it above the flow tag, but I found it unclear. We can say "before" but I think it's clear with "above".

Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
.. In the configuration XML, place your cursor before the opening `<flow>` tag, and type `http`.
```

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: good point. I'll create tabs!

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 11]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
...
+* *Template or Example Project*
++
+Import an existing integration template or example from Anypoint Exchange. 
++
+image::int-create-integration.png["Create integration project from an Exchange asset."]
++
+If you start with an asset from Exchange:
 
+.. xref:start-acb.adoc#login-ide[Log in to Anypoint Platform from the IDE] so that you can find assets.
+.. From the IDE, search for the template or example.
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
.. From the IDE, search for a template or for an example. 
```

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 12]
```diff
@@ -50,23 +52,61 @@ See xref:start-add-folders.adoc[].
...
+
+* *Template or Example Project*
++
+Import an existing integration template or example from Anypoint Exchange. 
++
+image::int-create-integration.png["Create integration project from an Exchange asset."]
++
+If you start with an asset from Exchange:
 
+.. xref:start-acb.adoc#login-ide[Log in to Anypoint Platform from the IDE] so that you can find assets.
```
Comment (i-piziali) [inline modules/ROOT/pages/int-create-integrations.adoc]: They do not need to log in to find assets, they can view public assets logged out. They only need to log in if they want to use a private asset. I would suggest rewording this to say something more like "log in if you would like to use private assets"

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 5 above)
Comment (i-piziali) [reply modules/ROOT/pages/int-create-integrations.adoc]: I think the distinction is important since they are two different asset types, and the only two asset types that they can search for and use in this feature

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 12 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: removing

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 5 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: I am changing the wording a bit to make it clear that they are different things.

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: done

File: modules/ROOT/pages/int-create-integrations.adoc [hunk 13]
```diff
@@ -50,23 +52,65 @@ See xref:start-add-folders.adoc[].
...
+[tabs]
+====
+*Start with an Empty Project*::
++
+--
+Click *Empty Project* to start an integration project from scratch, without any preconfigured components.
+
+image::int-create-integration-scratch.png["Create integration project from scratch.", width=70%] 
+--
+*Start with a Template or Example*::
```
Comment (dukesphere) [inline modules/ROOT/pages/int-create-integrations.adoc]:
```suggestion
*Start with a Project Template or Example*::
```

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 2 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: changed back to "before"

**Review by i-piziali — APPROVED** [review]

File: modules/ROOT/pages/int-create-integrations.adoc (hunk 1 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-create-integrations.adoc]: ohhh, lemme add it

**Review by chris-campbell9 — APPROVED** [review]
